less-discourse/README.md

```diff
@@ -0,0 +1,31 @@
+
+### Menu on the Discourse Forum
+
+The goal was to rebuild the exact same menu in both UI and on the Discourse Forum.
+Only a few icons change (to differenciate between the already-too-similar-ones on Discourse) and the HTML for the menu on Discourse had to be adapted.
```

Comment:
I don't think this section is useful, it's rather a PR comment than something useful for developers that want to change the sidemenu.